port-forward-fun.conf

```diff
@@ -1,19 +0,0 @@
-##################### Notes ########################
-#                                                  #
-#  So, you wanna run a single Node server, huh?    #
-#  Here's what the Apache server might look like   #
-#                                                  #
-#  This will assume an Express server is already   # 
-#                running on Port 3000              #
-#                                                  #
-####################################################
-
-<VirtualHost *:80>
-    ServerAdmin          admin@mycoolsite.com
-    ServerName           mycoolsite.com
-    ProxyPreserveHost    On
-    ProxyPass            "/" "http://127.0.0.1:3000/"
-    ProxyPassReverse     "/" "http://127.0.0.1:3000/"
-    ErrorLog             ${APACHE_LOG_DIR}/mycoolsite.log
```

Comment:
Logs! These come in really handy although for the most part, you hopefully won't need 'em.

You can name the logs anything you want but I tend to follow the convention of sitename.log to make things easy to keep track of.

This log in particular will contain any errors that occur in the site. Keep in mind though that since this is a Node application, Apache won't be able to tell you eg; error on line 15 of your Javascript since it doesn't have anything to do with serving that up.

By default, on an Ubuntu server at least, ${APACHE_LOG_DIR} is a placeholder for /var/log/apache2/, just not hardcoded in case you ever decide to change its location.